src/Rules/ArchRule.php

```diff
@@ -16,15 +17,25 @@ class ArchRule implements DSL\ArchRule
     /** @var string */
     private $because;
 
-    public function __construct(Specs $specs, Constraints $constraints, string $because)
+    /** @var array */
+    private $classesToBeExcluded;
+
+    public function __construct(Specs $specs, Constraints $constraints, string $because, array $classesToBeExcluded)
     {
         $this->thats = $specs;
         $this->shoulds = $constraints;
         $this->because = $because;
+        $this->classesToBeExcluded = $classesToBeExcluded;
     }
 
     public function check(ClassDescription $classDescription, Violations $violations): void
     {
+        $fqcn = FullyQualifiedClassName::fromString($classDescription->getFQCN());
+
+        if (\in_array($fqcn->toString(), $this->classesToBeExcluded)) {
```

Comment:
Here we can use wildcards also not only check for a single class right?
For example you can say that you want to exclude a folder like"`App\Domain\Foo\*`

What do you think @micheleorselli

Reply:
We can also move this check inside `allSpecsAreMatchedBy`

Reply:
Wildcards are fine, we should support them

Reply:
yup, it would be nice to support wildcard :-) 

I guess the check can stay here

Reply:
Ok I added a new method to support wildcard here.